src/librustdoc/html/highlight.rs

```diff
@@ -312,7 +312,7 @@ fn write_footer(out: &mut Buffer, playground_button: Option<&str>) {
 enum Class {
     Comment,
     DocComment,
-    Attribute,
+    Attr,
```

Comment:
Why updating the variant name? Only the CSS class should be enough.

Reply:
A few variants don't have the same name as the equivalent CSS class. I'd prefer if you kept it as is please.